### PR TITLE
direct to payment page when error message displayed and continue to payment button clicked…

### PR DESCRIPTION
**Pull request by aokunade**
BI-12138 - Shopping basket error message

### CODE REVIEW

File: src/controllers/basket.controller.ts [hunk 1]
```diff
@@ -45,15 +42,20 @@ export const render = async (req: Request, res: Response, next: NextFunction) =>
             } else {
                 return;
             }
-        }
+        } 
 
+        if (req.url == continueToPaymentPath) {
+                addAnotherDocumentUrl = `${BASKET_URL}`;
+                continueToPaymentUrl = `${await proceedToPayment(req, res, next)}`;
```
Comment (VaughanJackson123) [inline src/controllers/basket.controller.ts:49]: What is the value of `continueToPaymentUrl` that results from this assignment?

Comment (aokunade) [reply src/controllers/basket.controller.ts:49]: from the proceedToPayment method in the controler

**Comment by VaughanJackson123** [comment]
This does not seem to work for me when I test it locally. I get the `Sorry, there is a problem with the service` page when I click on the `Continue to payment` button.

**Comment by aokunade** [comment]
> This does not seem to work for me when I test it locally. I get the `Sorry, there is a problem with the service` page when I click on the `Continue to payment` button.

It does work for me locally like i said in all instances.

**Comment by VaughanJackson123** [comment]
> It does work for me locally like i said in all instances.

OK, that's good to know. As I say, for whatever reason in my environment it errors in the same way whether on this branch or `master`.  It might be good to get somebody else to try it in theirs?

File: src/controllers/basket.controller.ts (hunk 1 above)
Comment (VaughanJackson123) [reply src/controllers/basket.controller.ts:49]: The other question I have about that particular assignment is why are you evaluating the `await proceedToPayment` in that manner (in a string expression)?

**Comment by VaughanJackson123** [comment]
> OK, that's good to know. As I say, for whatever reason in my environment it errors in the same way whether on this branch or master.

This was a problem in my Tilt environment.

**Review by VaughanJackson123 — APPROVED** [review]

**Pull request merged**